package.json

```diff
@@ -1,144 +1,159 @@
+        "underscore": "^1.13.1",
+        "url-join": "^4.0.1",
+        "vis": "4.20.1",
+        "which": "^2.0.2",
+        "winston": "^3.3.3",
+        "winston-loki": "^6.0.1",
+        "winston-papertrail": "^1.0.5",
+        "winston-transport": "^4.4.0",
+        "wolfy87-eventemitter": "^5.2.9",
+        "yaml": "^1.10.2"
+    },
+    "devDependencies": {
+        "approvals": "^4.0.0-beta.1",
+        "aws-sdk-mock": "^5.1.0",
+        "chai": "^4.3.4",
+        "chai-as-promised": "^7.1.1",
+        "chai-http": "^4.2.1",
+        "codecov": "^3.8.2",
+        "copy-webpack-plugin": "^5.1.2",
+        "css-loader": "^3.6.0",
+        "deep-equal-in-any-order": "^1.1.4",
+        "eslint": "^7.27.0",
+        "eslint-config-prettier": "^8.3.0",
+        "eslint-formatter-github": "^1.0.11",
+        "eslint-plugin-import": "^2.23.3",
+        "eslint-plugin-jsdoc": "^35.0.0",
+        "eslint-plugin-node": "^11.1.0",
+        "eslint-plugin-promise": "^5.1.0",
+        "eslint-plugin-requirejs": "^4.0.1",
+        "eslint-plugin-sonarjs": "^0.7.0",
+        "eslint-plugin-unicorn": "^32.0.1",
+        "file-loader": "^5.1.0",
+        "lint-staged": "^11.0.0",
+        "mini-css-extract-plugin": "^0.9.0",
+        "mocha": "^8.4.0",
+        "mock-fs": "^5.0.0",
+        "monaco-editor-webpack-plugin": "^3.1.0",
+        "nock": "^12.0.3",
+        "nyc": "^15.1.0",
+        "optimize-css-assets-webpack-plugin": "^5.0.6",
+        "prettier": "2.3.2",
+        "pretty-quick": "^3.1.1",
+        "sass": "^1.34.0",
+        "sass-loader": "^10.2.0",
+        "sinon": "^11.1.1",
+        "supervisor": "^0.12.0",
+        "terser-webpack-plugin": "^2.3.8",
+        "url-loader": "^3.0.0",
+        "webpack": "^4.46.0",
+        "webpack-cli": "^3.3.12",
+        "webpack-dev-middleware": "^3.7.3",
+        "webpack-dev-server": "^3.11.2",
+        "webpack-manifest-plugin": "^2.2.0"
+    },
+    "mocha": {
+        "require": [
+            "esm",
+            "test/_setup"
+        ]
+    },
+    "scripts": {
+        "ci-lint": "eslint --format github .",
+        "ci-test": "nyc mocha --recursive",
+        "lint": "eslint .",
+        "lint-fix": "eslint --fix .",
+        "test": "mocha --recursive",
+        "check": "npm run lint && npm run test",
+        "dev": "cross-env NODE_ENV=DEV node -r esm app.js",
+        "debugger": "cross-env NODE_ENV=DEV node --inspect -r esm app.js --debug",
+        "debug": "cross-env NODE_ENV=DEV node -r esm app.js --debug",
+        "start": "webpack && cross-env NODE_ENV=LOCAL node -r esm app.js",
+        "codecov": "codecov --disable=gcov",
+        "sentry": "npx -p @sentry/cli sentry-cli",
+        "update-browserslist": "npx browserslist@latest -- --update-db",
+        "prepare": "husky install",
+        "format": "prettier --ignore-unknown",
+        "format-fix": "prettier --write --ignore-unknown"
+    },
+    "lint-staged": {
+        "*.{js,css}": [
+            "npm run ci-lint --",
```

Comment:
ESLint not automatically fixing the code is intended, as this way we keep the old behaviour, just in case